corehq/apps/data_dictionary/util.py

```diff
@@ -15,13 +20,13 @@ class OldExportsEnabledException(Exception):
 
 
 def generate_data_dictionary(domain):
-    properties = get_all_case_properties(domain)
+    properties = _get_all_case_properties(domain)
     _create_properties_for_case_types(domain, properties)
     CaseType.objects.filter(domain=domain, name__in=list(properties)).update(fully_generated=True)
     return True
 
 
-def get_all_case_properties(domain, case_types=None):
+def _get_all_case_properties(domain, case_types=None):
```

Comment:
(there's another function with the same name, so I figure this will make things less confusing)